src/dark-mode-toggle.mjs

```diff
@@ -126,7 +126,7 @@ input {
 }
 
 input:focus + label {
-  outline: rgb(229, 151, 0) auto 2px;
+  outline: #E59700 auto 2px;
```

Comment:
Agree, but we want them to be [lowercase](https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation).

Reply:
Done.